src/content/community/releases/_index.en.md

```diff
@@ -19,6 +19,7 @@ weight = 40
 * Fixed an issue with Service Discovery that caused a new `EndpointSlice` to be created when the labels on the exporting `Service`
   were updated.
 * New options were added to `subctl cloud prepare` to support a custom vpc for AWS.
+* Fixed an issue where AWS cloud prepare failed to find the VPC.
```

Comment:
I think you intended this for L16 above.

Reply:
oh sorry, I got it wrong. Fixed it.